tests/server/providers/jellyfin/test_parsers.py

```diff
@@ -0,0 +1,71 @@
+"""Test we can parse Jellyfin models into Music Assistant models."""
+
+import logging
+import pathlib
+from collections.abc import AsyncGenerator
+
+import aiofiles
+import aiohttp
+import pytest
+from aiojellyfin import Artist, Connection, SessionConfiguration
+from mashumaro.codecs.json import JSONDecoder
+from syrupy.assertion import SnapshotAssertion
+
+from music_assistant.server.providers.jellyfin.parsers import parse_album, parse_artist, parse_track
+
+FIXTURES_DIR = pathlib.Path(__file__).parent / "fixtures"
+ARTIST_FIXTURES = list(FIXTURES_DIR.glob("artists/*.json"))
+ALBUM_FIXTURES = list(FIXTURES_DIR.glob("albums/*.json"))
+TRACK_FIXTURES = list(FIXTURES_DIR.glob("tracks/*.json"))
+
+ARTIST_DECODER = JSONDecoder(Artist)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture
+async def connection() -> AsyncGenerator[Connection, None]:
+    """Spin up a dummy connection."""
+    async with aiohttp.ClientSession() as session:
```

Comment:
do we want to globally mock the aiohttp session ?

Reply:
To be honest i'm a bit annoyed this bit needs a ClientSession at all. its not actually used by the test, I just need one to instance the aiojellyfin objects used in the test. I am probably going to try to refactor it away at some point.

I am playing around with mocking ClientSession generally in another branch. The idea is to provide a fake for the API in aiojellyfin that we can load fixtures into, then have some way to use that fake in the MA tests with the `aiohttp_client` fixture (which does have its own ClientSession). I'll probably combine that with something like the socket blocker from HA's tests.